postgresql/postgresql.rb

```diff
@@ -293,18 +293,13 @@ def monitor_postgresql(pg_servers, group_name)
     pg_servers.each do |mhost|
       return if @interrupted
 
-      thishash = server_db_map[mhost['hostname']]
       pg_dbs = mhost['databases']
       pg_dbs.each do |db|
         return if @interrupted
```

Comment:
add appropriate new line to make code look clean (atleast around methods you are changing)